JeMPI_Apps/JeMPI_UI/src/services/apiRoutes.ts

```diff
@@ -11,13 +11,18 @@ const ROUTES = {
   POST_UPDATE_NOTIFICATION: '/notificationRequest',
   POST_SIMPLE_SEARCH: '/search',
   POST_CUSTOM_SEARCH: '/customSearch',
```

Comment:
@harmochiky please remove these old endpoints and update the mockbackend dependency